GTG/gtk/browser/browser.py

```diff
@@ -117,7 +130,7 @@ def __init__(self, requester, vmanager):
         self.activetree.register_cllbck('node-deleted-inview',
                                         self._update_window_title)
         self._update_window_title()
```

Comment:
Do you mind (in a separate commit/PR) remove the whole madness of listing # active tasks in the title of the window? Somebody (can't find the mail) was horrified about the  usage of window title for counting tasks when you have the left pane with the numbers. It should be set in .ui to a sane value and probably shouldn't be tinkered with during runtime.